install/linux/usr/share/odemis/sim/meteor-sim.odm.yaml

```diff
@@ -0,0 +1,198 @@
+# For now, this is esssentially just a FM microscope, as the FIB part is handled
+# separately by the SEM
+
+METEOR-Sim: {
+    class: Microscope,
+    role: meteor,
+    children: ["Optical Objective",
+               "Optical Focus",
+               "Light Source", "Filter Wheel", "Camera",
+               "Linked Stage", "EBeam",
+    ],
+}
+
+"SEM": {
+    class: simsem.SimSEM,
+    role: null,
+    init: {
+           image: "simsem-fake-output.h5", # any large 16 bit image is fine
+    },
+    children: {scanner: "EBeam",
+               detector0: "SE Detector",
+               focus: "EBeam Focus",
+    }
+}
+
+"EBeam": {
+    # Internal child of SimSEM, so no class
+    role: e-beam,
+    init: {},
+    affects: ["BSED Phenom", "Camera"], # affects the CCD in case of cathodoluminescence
+}
+
+"EBeam Focus": {
+    role: ebeam-focus,
+    init: {},
+    affects: ["EBeam"]
+}
+
+"SE Detector": {
+    # Internal child of SimSEM, so no class
+    role: se-detector,
+    init: {},
+}
+
+# Normally provided by the SEM
+"Stage": {
+    class: tmcm.TMCLController,
+    role: stage-bare,
+    init: {
+        port: "/dev/fake6",
+        address: null,
+        axes: ["x", "y", "z", "rx", "rz"],
+        ustepsize: [1.e-7, 1.e-7, 1.e-7, 1.2e-6, 1.2e-6], # unit/µstep
+        rng: [[-0.1, 0.1], [-0.05, 0.05], [0, 0.1], [-2, 2], [0, 6.28]],
+        unit: ["m", "m", "m", "rad", "rad"],
+        refproc: "Standard",
+    },
+    metadata: {
+        FAV_POS_DEACTIVE: {'rx': 0, 'rz': 0, 'x':  0.035, 'y':  0.0042, 'z':  -0.0068},
+        SEM_IMAGING_RANGE: {"x": [0.04, 0.06], "y": [-0.001, 0.003], "z": [0.000, 0.004]},
+        FM_IMAGING_RANGE: {"x": [0.07, 0.09], "y": [0.004, 0.008], "z": [0.0, 0.004]},
+        SAMPLE_CENTERS: {"GRID 1": {'x':  0.05, 'y':  0.001, 'z': 0.002}, "GRID 2": {'x': 0.0534, 'y': 0.001, 'z': 0.002}},
+        GRID_SHIFT: {"x": 0.0034},
```

Comment:
We don't need GRID_SHIFT. When switching to another grid, use the SAMPLE_CENTERS info.